chevah/compat/tests/normal/test_system_users.py

```diff
@@ -40,7 +40,7 @@ def test_getHomeFolder_posix(self):
         Check getHomeFolder on Linux and Unix.
         """
         home_folder = system_users.getHomeFolder(
-            username=mk.username)
+            username=mk.username).rstrip('/')
```

Comment:
We should not fix the test, but rather update the implementation so that getHomeFolder will always return the path without the trailing slash.

I will make the change now. Thanks!

Reply:
I beg to differ, but it's up to you how to fix it… My problem is your change has brought the issue back, see for example https://chevah.com/buildbot/builders/compat-ubuntu-1604/builds/5